### PR TITLE
Add liquidity message.

### PR DESCRIPTION
**Pull request by biocom**
For ticket #867

- Added message with a link to the Balances (wallet) page.
- Made the inputfield have a white background. In my opinion the grey inputfield inside the grey container wasn't clear.

Screenshot 2020-04-14 at 11 21 30 

Screenshot 2020-04-14 at 11 21 37

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr885--dexreact.review.gnosisdev.com

**Comment by W3stside** [comment]
> Great! Much clearer now.
> 
> Regarding the message, Kei gave us some feedback and changed the message slightly. Would you change it before merge to:
> 
> `Your liquidity is equal to the amount you have deposited into your exchange wallet. Be sure to deposit at least one stablecoin on the Balances tab.`

I would say:
`Your liquidity is equal to the amount you have deposited into your exchange wallet. Be sure to deposit at least one stablecoin via the Balances tab.`

File: src/components/PoolingWidget/DefineSpread.styled.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export const DefineSpreadWrapper = styled(InputBox)`
     width: 100%;
     height: 5.6rem;
     padding: 0 1rem;
+    background: white;
```
Comment (W3stside) [inline src/components/PoolingWidget/DefineSpread.styled.ts:18]: can you make this `var(...)` for dark mode

**Comment by biocom** [comment]
Updated message:
Screenshot 2020-04-14 at 12 00 50

File: src/components/PoolingWidget/DefineSpread.styled.ts (hunk 1 above)
Comment (biocom) [reply src/components/PoolingWidget/DefineSpread.styled.ts:18]: Ah, sorry forgot to change, will do now.

File: src/components/PoolingWidget/index.tsx [hunk 2]
```diff
@@ -272,6 +291,16 @@ const PoolingInterface: React.FC = () => {
               <SubComponents step={step} {...restProps} />
             </ContentWrapper>
 
+            <LiquidityMessage>
+              <p>
+                Your liquidity is equal to the amount you have deposited into your exchange wallet.
+                <br />{' '}
```
Comment (Velenir) [inline src/components/PoolingWidget/index.tsx:297]: Why `{' '}`? It gets collapsed and you don't see it anyway

File: src/components/PoolingWidget/index.tsx [hunk 3]
```diff
@@ -272,6 +291,16 @@ const PoolingInterface: React.FC = () => {
...
+                <b>
+                  Be sure to deposit at least one stablecoin on the <a href="/wallet">Balances</a> tab.
```
Comment (Velenir) [inline src/components/PoolingWidget/index.tsx:299]:
```suggestion
                  Be sure to deposit at least one stablecoin on the <Link to="/wallet">Balances</Link> tab.
```
use router, don't reload page

File: src/components/PoolingWidget/index.tsx (hunk 2 above)
Comment (biocom) [reply src/components/PoolingWidget/index.tsx:297]: Think it was added by the yarn lint fix. Can remove.

File: src/components/PoolingWidget/index.tsx (hunk 3 above)
Comment (biocom) [reply src/components/PoolingWidget/index.tsx:299]: Thanks, will update. I think we should also do it in the deposit message (in the trade form).

**Comment by biocom** [comment]
Updated to use React router on two occasions (deposit and liquidity page).

**Comment by biocom** [comment]
Thanks for the reviews!

**Pull request merged**